src/components/sidebar/SidebarHeader/index.tsx

```diff
@@ -96,12 +96,22 @@ const SafeHeader = (): ReactElement => {
 
           <Track {...OVERVIEW_EVENTS.COPY_ADDRESS}>
             <CopyButton text={addressCopyText} className={css.iconButton}>
-              <SvgIcon component={CopyIconBold} inheritViewBox color="primary" fontSize="small" />
+              <SvgIcon
+                component={CopyIconBold}
+                inheritViewBox
+                color="primary"
+                fontSize="small"
+                sx={{
+                  '& path': {
+                    fill: ({ palette }) => palette.primary.main,
+                  },
+                }}
```

Comment:
Another way to achieve this for SVGs is set their fill color to `currentColor` so that we can always change it from the parent via the `color` attribute.